changelog/261.improvement.md

```diff
@@ -0,0 +1,2 @@
+Under each form of the domain, the definition of slot mappings without the keyword `required_slots` is now deprecated. Please always use the 
```

Comment:
If I put myself in the shoe of a user I find this a tiny bit cryptic. How about something like: "Adding support for forms with the `required_slots` key which is introduced as part of Rasa Open Source 2.6."?